.changeset/smart-cameras-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/vue': major
+---
+
+chore: improve app setup call
```

Comment:
Yes.

`astro.config.mjs`:
```js
  integrations: [
    vue({
      appEntrypoint: '/src/pages/_app',
      template: {
        compilerOptions: {
          //
        },
      },
    }),
  ],
```

In the `/src/pages/_app.js` file, if there is no export defualt, it may cause errors.